servlet/src/main/java/com/linecorp/armeria/server/servlet/AbstractChannelHandler.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.server.servlet;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.netty.channel.ChannelDuplexHandler;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelPromise;
+import io.netty.handler.timeout.IdleStateEvent;
+import io.netty.util.internal.TypeParameterMatcher;
+
+/**
+ *  An abstract netty ChannelHandler.
+ */
+public abstract class AbstractChannelHandler<I,O> extends ChannelDuplexHandler {
```

Comment:
Because this class is not used, could you remove this?

Reply:
Totally agree, I will remove this.